pipeline/plots/plot_functions.jl

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
I think I would put these in the analysis module as this is part of the analysis and we don't plan to ship it? Alternatively I suppose we could have them as part of a less generic paper module or something (and so the pipeline would aim to be generic and the paper would aim to have the specific implementations for this work in it?) that might be a bit extra but also might not be that much work to get some quite clean code?